AudioChunk audio_chunk
AudioConfigM audio_config
time start_time
float64 duration
int64 utterance_index
int64 audio_chunk_index
bool is_end
